etc/systemd/system/duffy-metaclient.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+After=network.target
+Requires=duffy-metaclient.service
```

Comment:
Uhm… :wink:

```
Requires=duffy-app.service
```

Just had a thought: having our service units depend on each other kinda presumes that they run on the same machine, which they don't have to: we can point the metaclient to any other URL for the app, and both app and task worker could live on different machines connected via a redis service (on yet another machine :thinking:). So, let's add comments to the `Required=`, `RequiredBy=` lines à la "comment out if {metaclient,app} and {app,task worker} don't run on the same machine"?